flink-table/flink-sql-client/conf/sql-client-defaults.yaml

```diff
@@ -100,18 +100,10 @@ execution:
   planner: blink
   # 'batch' or 'streaming' execution
   type: streaming
-  # allow 'event-time' or only 'processing-time' in sources
-  time-characteristic: event-time
-  # interval in ms for emitting periodic watermarks
-  periodic-watermarks-interval: 200
   # 'changelog', 'table' or 'tableau' presentation of results
   result-mode: table
   # maximum number of maintained rows in 'table' presentation of results
   max-table-result-rows: 1000000
-  # parallelism of the program
-  parallelism: 1
```

Comment:
What about only comment out (not remove) the `parallelism` and `restart-strategy`?

1. User can still easily enable them in the `sql-client-defaults.yaml`.
2. It seems that other options (`time-characteristic`, `periodic-watermarks-interval`, `max-parallelism`) override the default value of underlying pipeline configuraiton. So removing them may affect the SQL job behavior. 

Besides 
We may also need to update `org.apache.flink.table.client.config.entries.ExecutionEntry#getParallelism`, it hard code to return a default value 1.